kivy/properties.pyx

```diff
@@ -1699,53 +1773,54 @@ cdef class VariableListProperty(Property):
         # reset here, it'll be changed in parse is we use anything that is not px
         ps.uses_scaling = 0
         try:
-            if isinstance(x, (list, tuple)):
+            if tp is int or tp is long or tp is float or isinstance(x, str):
```

Comment:
Why not testing with `type(x) is str` instead of `isinstance(x, str)` when former is faster?
Type `long` is merged with `int` in Python3, so can we remove all `long` occurrences?

Reply:
I guess you're right and I overlooked it. See https://github.com/kivy/kivy/pull/7439.